### ISSUE TITLE
Your nightstick passes through the grey death spirit with no effect!

### ISSUE THREAD

**Issue by drgabellia** [rpherbig/dr-scripts#528]
Please have combat script recognize when weapon needs to be blessed when hunting undead.

Currently script will prompt for repeated swings with the unblessed weapon, then switch to another weapon, with no skill being learned with the unblessed one.

Message with unblessed weapon:  Your nightstick passes through the grey death spirit with no effect!

**Comment by rcuhljr** [comment]
So did it not cast bless when switching to the nightstick? Did you have bless defined as one of your buffs?

If you happen to get a log of the problem (switch to nightstick, not cast bless, attack ten times) it would be helpful.

**Comment by drgabellia** [comment]
Yes, bless is one of my buffs.   Eventually it DID bless my nightstick.  However, I'm wondering if it might be able to bless the nightstick (or any weapon) as soon as the message is received?

* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're nimbly balanced and overwhelming your opponent.]
You lower your shoulders and begin to twirl your nightstick.
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're nimbly balanced and overwhelming your opponent.]
You feel fully prepared to cast your spell.
Your nightstick passes through the grey death spirit with no effect!
[Roundtime 2 sec.]
Making an agile leap, you unleash your twirling nightstick upon a pale grey death spirit!

Your nightstick lands a glancing (0/23) strike to a pale grey death spirit's abdomen!

[You're winded, nimbly balanced and in dominating position.]
Roundtime: 5 sec.
* A pale grey death spirit claws at you.  You fend off nearly all the claw with a steel nightstick.  
[You're winded, nimbly balanced with no advantage.]
* A pale grey death spirit claws at you.  You evade.  
[You're winded, nimbly balanced and in dominating position.]
* A pale grey death spirit pounds at you.  You badly fail to block with a steel nightstick.  The fist lands a light hit (1/23) to your abdomen.
[You're bruised, winded, nimbly balanced and in dominating position.]
You feel fully attuned to the mana streams again.
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced and have slight advantage.]
* A pale grey death spirit pounds at you.  You evade.  
[You're bruised, nimbly balanced and in very strong position.]
* A pale grey death spirit claws at you.  You counter nearly all the claw with a steel nightstick.  
[You're bruised, nimbly balanced and in superior position.]

[combat-trainer]>cast

You gesture.
Your armband emits a loud *snap* as it discharges all its power to aid your spell.
A bright silver nimbus roars up around you as streaks of steel-blue light descend in a convex field.
A sense of calm focus takes hold as the luminous pattern washes over you.  You feel slightly more able to avoid attacks with your energized joints.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're bruised, winded, nimbly balanced and in superior position.]
Roundtime: 3 sec.

* A pale grey death spirit claws at you.  You repulse nearly all the claw with a steel nightstick.  
[You're bruised, winded, nimbly balanced and in excellent position.]

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You beat off nearly all the claw with a steel nightstick.  
[You're bruised, winded, nimbly balanced and in very strong position.]

[combat-trainer]>pre Centering 20

That won't affect your current attunement very much.
You begin chanting a prayer to invoke the Centering spell.

[combat-trainer]>charge my armband 25

You harness a moderate amount of energy and attempt to channel it into your armband.
You are able to channel all the energy into the armband.
The armband absorbs all of the energy.
Roundtime: 4 sec.
* A pale grey death spirit pounds at you.  You deflect nearly all the fist with a steel nightstick.  
[You're bruised, nimbly balanced and in strong position.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're bruised, nimbly balanced and in superior position.]

[combat-trainer]>charge my armband 25

You harness a moderate amount of energy and attempt to channel it into your armband.
You are able to channel all the energy into the armband.
The armband absorbs all of the energy.
Roundtime: 4 sec.
* A pale grey death spirit pounds at you.  You evade.  
[You're bruised, nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You evade.  
[You're bruised, nimbly balanced and in very strong position.]

[combat-trainer]>hunt

You take note of all the tracks in the area, so that you can hunt anything nearby down.

You were unable to locate any followable tracks.

Roundtime: 8 sec.
* A pale grey death spirit pounds at you.  You dodge.  
[You're bruised, nimbly balanced and in very strong position.]
A pale grey death spirit appears less distracted.
* A pale grey death spirit pounds at you.  You counter nearly all the fist with a steel nightstick.  
[You're bruised, nimbly balanced and in better position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced with no advantage.]

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 1 sec.]

[combat-trainer]>invoke my armband

The armband pulses with Holy energy.  You reach for its center and forge a magical link to it, readying all of its mana for your use.
Roundtime: 1 sec.
You feel fully rested.
* A pale grey death spirit slams down at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced and in strong position.]

[combat-trainer]>circle

You fake a pale grey death spirit, first moving one way and then another, leaving it off balance.
[You're bruised, nimbly balanced and in very strong position.]
Roundtime: 3 sec.
You feel fully prepared to cast your spell.
* A pale grey death spirit claws at you.  You dodge.  
[You're bruised, nimbly balanced and in superior position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced and in strong position.]

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 3 sec.]
You feel fully rested.

[combat-trainer]>cast

You gesture.
Your armband emits a loud *snap* as it discharges all its power to aid your spell.
An odd tingling sensation spreads upward from your feet through your entire body.

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're bruised, winded, nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, winded, nimbly balanced and in excellent position.]
* A pale grey death spirit claws at you.  You knock aside nearly all the claw with a steel nightstick.  
[You're bruised, winded, nimbly balanced and in superior position.]

[combat-trainer]>pre Bless 20

That won't affect your current attunement very much.
You begin chanting a prayer to invoke the Bless spell.

[combat-trainer]>charge my armband 25

You harness a moderate amount of energy and attempt to channel it into your armband.
You are able to channel all the energy into the armband.
The armband absorbs all of the energy.
Roundtime: 4 sec.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced and in dominating position.]
* A pale grey death spirit claws at you.  You beat off nearly all the claw with a steel nightstick.  
[You're bruised, nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You dodge.  
[You're bruised, nimbly balanced and in dominating position.]

[combat-trainer]>charge my armband 25

You harness a moderate amount of energy and attempt to channel it into your armband.
You are able to channel all the energy into the armband.
The armband absorbs all of the energy.
Roundtime: 4 sec.

[hunting-buddy: ***STATUS*** 178 minutes of hunting remaining or waiting on Large Edged, Staves, Tactics, Heavy Thrown, Brawling]


[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, winded, nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, winded, nimbly balanced and in good position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, winded, nimbly balanced and in strong position.]

[combat-trainer]>invoke my armband

The armband pulses with Holy energy.  You reach for its center and forge a magical link to it, readying all of its mana for your use.
Roundtime: 1 sec.

[combat-trainer]>circle

You fake a pale grey death spirit, first moving one way and then another, leaving it off balance.
[You're bruised, winded, nimbly balanced and in superior position.]
Roundtime: 4 sec.

[combat-trainer]>maneuver twirl

You must rest a bit longer before attempting that maneuver again.
* A pale grey death spirit claws at you.  You turn aside nearly all the claw with a steel nightstick.  
[You're bruised, nimbly balanced and in excellent position.]
* A pale grey death spirit claws at you.  You beat off nearly all the claw with a steel nightstick.  
[You're bruised, nimbly balanced and in excellent position.]

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced and in strong position.]

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and in dominating position.]
* A pale grey death spirit pounds at you.  You turn aside nearly all the fist with a steel nightstick.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>attack

Your nightstick passes through the grey death spirit with no effect!
[Roundtime 1 sec.]

[combat-trainer]>stow my nightstick

You put your nightstick in your kidskin baldric.
You feel fully prepared to cast your spell.

**Comment by rcuhljr** [comment]
You might just try moving bless up to the top of your buff list. This seems like it's just a problem when you start a hunt.

**Comment by drgabellia** [comment]
will try that, thank you!

**Comment by drgabellia** [comment]
The snippet below happened while I had all my buffs up.  As you can see, the hhr'ata was lobbed 30+ times while not blessed.  I finally snapped cast a bless at it manually.

You get a steel hhr'ata from inside your haversack.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata


The air suddenly swirls with shadows as two sanguine eyes materialize out of thin air to coalesce into the form of a pale grey death spirit!
The grey death spirit begins to advance on you!
The grey death spirit advances from nearby and is closing steadily.
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You knock aside nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and overwhelming your opponent.]
* A pale grey death spirit claws at you.  You repulse nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're winded, adeptly balanced and overwhelming opponent.]
Roundtime: 4 sec.
The grey death spirit closes to melee range on you!

[Derium] "So, a normal day?"
A cage of shadowy tendrils coalesces around a pale grey death spirit.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You counter nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're winded, nimbly balanced and in dominating position.]
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in better position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in superior position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You dodge.  
[You're winded, nimbly balanced and overwhelming your opponent.]
You pick up a steel hhr'ata.

[combat-trainer]>rub my Glythtide orb

As you rub the Glythtide orb, it glows slightly more intensely and you feel a strange tugging, as if something has been moved from you to the orb.  You sense, though, that your sacrifice is not yet fully prepared.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in good position.]
* A pale grey death spirit claws at you.  You deflect nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're winded, nimbly balanced and in very strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You evade.  
[You're winded, nimbly balanced and in dominating position.]
A pale grey death spirit appears less distracted.
* A pale grey death spirit claws at you.  You evade.  
[You're winded, nimbly balanced and have slight advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're winded, nimbly balanced and in dominating position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in excellent position.]
You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're winded, nimbly balanced and in dominating position.]
Roundtime: 3 sec.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in very strong position.]
* A pale grey death spirit slams down at you.  You dodge.  
[You're winded, nimbly balanced and overwhelming your opponent.]
* A pale grey death spirit claws at you.  You dodge.  
[You're winded, nimbly balanced and in dominating position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You dodge.  
[You're winded, nimbly balanced and in excellent position.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're winded, nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You dodge.  
[You're winded, nimbly balanced and in dominating position.]
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in strong position.]
A pale grey death spirit appears less distracted.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're winded, nimbly balanced and in dominating position.]
Roundtime: 4 sec.
A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit slices wide at you.  You dodge.  
[You're winded, nimbly balanced and in strong position.]
* A pale grey death spirit pounds at you.  You counter nearly all the fist with a steel hhr'ata.  
[You're winded, nimbly balanced and in dominating position.]

[hunting-buddy: ***STATUS*** 110 minutes of hunting remaining or waiting on Large Edged, Staves, Tactics, Heavy Thrown, Brawling, Light Armor]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're winded, nimbly balanced and in superior position.]
A pale grey death spirit points at you and wails!

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>hunt

You take note of all the tracks in the area, so that you can hunt anything nearby down.

You were unable to locate any followable tracks.

Roundtime: 8 sec.
You feel fully attuned to the mana streams again.
* A pale grey death spirit pounds at you.  You dodge.  
[You're winded, nimbly balanced and overwhelming your opponent.]
* A pale grey death spirit pounds at you.  You turn aside nearly all the fist with a steel hhr'ata.  
[You're winded, nimbly balanced and in superior position.]
* A pale grey death spirit pounds at you.  You evade.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You dodge.  
[You're winded, nimbly balanced and have slight advantage.]
* A pale grey death spirit pounds at you.  You evade.  
[You're winded, nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully attuned to the mana streams again.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit claws at you.  You turn aside nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and in superior position.]
* A pale grey death spirit pounds at you.  You evade.  
[You're nimbly balanced and in superior position.]
You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're nimbly balanced and in dominating position.]
Roundtime: 3 sec.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit slices wide at you.  You dodge.  
[You're nimbly balanced and in good position.]
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and in excellent position.]
* A pale grey death spirit pounds at you.  You evade.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>perc

You reach out with your senses and see glowing (11/21) streams of golden Holy mana radiating through the area.
Letting your senses extend further, you feel there is shimmering (9/21) mana to the south, and luminous (14/21) mana to the north.
You sense the Centering spell upon you, which will last for about twenty-five roisaen.
You sense the Benediction spell upon you, which will last for about twenty-one roisaen.
You sense your Osrel Meraud spell linking you to a luminous orb of coruscating rainbowed light, which should last until you release it.
You sense the Persistence of Mana spell upon you, which is linked to your Osrel Meraud orb.
You sense the Murrula's Flames spell upon you, which is linked to your Osrel Meraud orb.
You sense the Glythtide's Gift spell upon you, which is linked to your Osrel Meraud orb.
You sense the Major Physical Protection spell upon you, which will last for about twenty roisaen.
You sense the Bless spell upon you, which will last for about nineteen roisaen.
You sense the Hydra Hex spell upon you, which will last until you fail to provide 15 mana for it.
You sense the Protection from Evil spell upon you, which will last for about twenty-three roisaen.
Roundtime: 3 sec.
You feel fully rested.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You counter nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit slams down at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in better position.]
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in strong position.]
You feel fully attuned to the mana streams again.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

A cage of shadowy tendrils coalesces around a pale grey death spirit.
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit pounds at you.  You deflect nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit claws at you.  You knock aside nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in dominating position.]
You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're winded, nimbly balanced and overwhelming opponent.]
Roundtime: 3 sec.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully attuned to the mana streams again.
A pale grey death spirit speaks in a language older than time.
* A pale grey death spirit slams down at you.  You turn aside nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
Like the trembling illusion of heat haze, a shimmer runs over your footflower.  The mirage is gone as quickly as it appeared.
* A pale grey death spirit pounds at you.  You evade.  
[You're winded, nimbly balanced and in good position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
A white glow flares around you.
* A pale grey death spirit pounds at you.  You fail to evade.  The fist lands a grazing (0/23) strike to your right arm.
[You're winded, nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and opponent has slight advantage.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

< You lob a steel hhr'ata at a pale grey death spirit.  A pale grey death spirit dodges.  
The hhr'ata hits a wall and falls to the floor!
[You're winded, nimbly balanced and in superior position.]
[Roundtime 2 sec.]
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in dominating position.]

[combat-trainer]>get my hhr'ata

You pick up the hhr'ata lying at your feet.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[hunting-buddy: ***STATUS*** 109 minutes of hunting remaining or waiting on Large Edged, Staves, Tactics, Heavy Thrown, Brawling, Light Armor]

* A pale grey death spirit slams down at you.  You slap away nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're winded, nimbly balanced and in good position.]
A pale grey death spirit appears less distracted.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're nimbly balanced and in dominating position.]
Roundtime: 4 sec.
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in good position.]
A pale grey death spirit speaks in a language older than time.
A spectral, darkly shimmering hydra rears high behind you, and with a discordant roar lashes out in an unbridled turmoil of thirteen long-necked heads thrashing about!

A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.
The mirage-hydra sinks into the ground.


[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in better position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You beat off nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in better position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>infuse om 15

You tap into the mana from fifteen of the surrounding streams and attempt to keep it channeling in a stream around you.
Roundtime: 1 sec.
You concentrate intently, attempting to infuse a significant amount of your harnessed mana into your active Osrel Meraud spell pattern.
Roundtime: 1 sec.
You sense that you were able to channel about two-thirds of your infusion's power into a luminous orb of coruscating rainbowed light.  A faint tugging sensation emanates from the orb, as if it hungers for more.
* A pale grey death spirit claws at you.  You counter nearly all the claw with a steel hhr'ata.  
[You're nimbly balanced and in superior position.]
A pale grey death spirit speaks in a language older than time.

[combat-trainer]>infuse om 15

You tap into the mana from fifteen of the surrounding streams and attempt to keep it channeling in a stream around you.
Roundtime: 1 sec.
You concentrate intently, attempting to infuse a significant amount of your harnessed mana into your active Osrel Meraud spell pattern.
Roundtime: 1 sec.
You sense that you were able to channel about two-thirds of your infusion's power into a luminous orb of coruscating rainbowed light.  A faint tingle washes through you as some of the energy is diffused, the orb having reached its full capacity.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and have slight advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in dominating position.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're nimbly balanced with no advantage.]
* A pale grey death spirit slams down at you.  You beat off nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in dominating position.]
You pick up a steel hhr'ata.

[combat-trainer]>hunt

You take note of all the tracks in the area, so that you can hunt anything nearby down.

You were unable to locate any followable tracks.

Roundtime: 8 sec.
* A pale grey death spirit slams down at you.  You evade.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit pounds at you.  You dodge.  
[You're nimbly balanced and overwhelming your opponent.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in better position.]
You feel fully rested.
>get card

[combat-trainer]>lob

You pick up a Guildleader Karazhil card.
Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You repulse nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in good position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're winded, nimbly balanced and in dominating position.]
Roundtime: 4 sec.
* A pale grey death spirit claws at you.  You turn aside nearly all the claw with a steel hhr'ata.  
[You're winded, nimbly balanced and overwhelming your opponent.]
* A pale grey death spirit claws at you.  You counter nearly all the claw with a steel hhr'ata.  
[You're winded, nimbly balanced and in better position.]
You feel fully attuned to the mana streams again.
>put card in bald
You put your card in your kidskin baldric.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit pounds at you.  You repulse nearly all the fist with a platinum parry stick inlaid with bands of jade.  
[You're winded, nimbly balanced and have slight advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.
* A pale grey death spirit claws at you.  You dodge.  
[You're winded, nimbly balanced and in dominating position.]

[combat-trainer]>lob

* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in better position.]
Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit claws at you.  You slap away nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in good position.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and in dominating position.]
* A pale grey death spirit slices wide at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and have slight advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You repulse most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and opponent has slight advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
A pale grey death spirit appears less distracted.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in very strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're solidly balanced with no advantage.]
You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're nimbly balanced and in very strong position.]
Roundtime: 3 sec.

[hunting-buddy: ***STATUS*** 108 minutes of hunting remaining or waiting on Large Edged, Staves, Tactics, Heavy Thrown, Brawling, Light Armor]


[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit claws at you.  You fend off most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're winded, nimbly balanced with no advantage.]
A pale grey death spirit whispers to itself.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in better position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

< You lob a steel hhr'ata at a pale grey death spirit.  A pale grey death spirit dodges.  
The hhr'ata hits a wall and falls to the floor!
[You're winded, nimbly balanced and in superior position.]
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're winded, nimbly balanced and have slight advantage.]

[combat-trainer]>get my hhr'ata

You pick up the hhr'ata lying at your feet.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
You feel fully attuned to the mana streams again.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

* A pale grey death spirit claws at you.  You turn aside nearly all the claw with a steel hhr'ata.  
[You're nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in better position.]
Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and have slight advantage.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
A pale grey death spirit appears less distracted.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're nimbly balanced and in dominating position.]
Roundtime: 3 sec.
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced and in better position.]
You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You repulse most of the claw with a steel hhr'ata.  
[You're nimbly balanced and in better position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit slices wide at you.  You beat off nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in strong position.]
You pick up a steel hhr'ata.

[combat-trainer]>perc

You reach out with your senses and see glowing (11/21) streams of golden Holy mana radiating through the area.
Letting your senses extend further, you feel there is shimmering (9/21) mana to the south, and luminous (14/21) mana to the north.
You sense the Centering spell upon you, which will last for about twenty-three roisaen.
You sense the Benediction spell upon you, which will last for about nineteen roisaen.
You sense your Osrel Meraud spell linking you to a luminous orb of coruscating rainbowed light, which should last until you release it.
You sense the Persistence of Mana spell upon you, which is linked to your Osrel Meraud orb.
You sense the Murrula's Flames spell upon you, which is linked to your Osrel Meraud orb.
You sense the Glythtide's Gift spell upon you, which is linked to your Osrel Meraud orb.
You sense the Major Physical Protection spell upon you, which will last for about eighteen roisaen.
You sense the Bless spell upon you, which will last for about seventeen roisaen.
You sense the Hydra Hex spell upon you, which will last until you fail to provide 15 mana for it.
You sense the Protection from Evil spell upon you, which will last for about twenty-one roisaen.
Roundtime: 3 sec.
You feel fully rested.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in strong position.]
* A pale grey death spirit claws at you.  You turn aside nearly all the claw with a steel hhr'ata.  
[You're nimbly balanced and in dominating position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in excellent position.]
* A pale grey death spirit claws at you.  You slap away most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in good position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in dominating position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're nimbly balanced and in dominating position.]
Roundtime: 3 sec.
You've gained a new rank in your understanding of combat tactics.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in strong position.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced with no advantage.]
* A pale grey death spirit claws at you.  You fend off nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>hunt

You take note of all the tracks in the area, so that you can hunt anything nearby down.

You were unable to locate any followable tracks.

Roundtime: 8 sec.
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and in good position.]
You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and have slight advantage.]
A white glow flares around you.
* A pale grey death spirit claws at you.  You barely fail to block with targe.  The claw lands a light hit (1/23) to your back.
[You're bruised, nimbly balanced and in good position.]

[combat-trainer]>lob

< You lob a steel hhr'ata at a pale grey death spirit.  A pale grey death spirit dodges.  
The hhr'ata hits a wall and falls to the floor!
[You're bruised, nimbly balanced and in good position.]
[Roundtime 3 sec.]
Your flower spider begins to weave a delicate web of shimmering glitter that encircles your left ankle.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're bruised, nimbly balanced and in excellent position.]
You feel fully rested.
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced with opponent in better position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in very strong position.]

[combat-trainer]>get my hhr'ata

You pick up the hhr'ata lying at your feet.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[hunting-buddy: ***STATUS*** 107 minutes of hunting remaining or waiting on Large Edged, Staves, Tactics, Heavy Thrown, Brawling, Light Armor]

A pale grey death spirit appears less distracted.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're nimbly balanced and in superior position.]
Roundtime: 3 sec.
You feel fully attuned to the mana streams again.
A pale grey death spirit speaks in a language older than time.
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in excellent position.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced with no advantage.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
You feel fully rested.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit pounds at you.  You block with a damite targe.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in strong position.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit pounds at you.  You block with a damite targe.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit claws at you.  You beat off most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and have slight advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You knock aside nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in very strong position.]
You feel fully rested.
You feel fully attuned to the mana streams again.

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced with no advantage.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in superior position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're nimbly balanced and in dominating position.]
Roundtime: 3 sec.
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in better position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and in dominating position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced and in strong position.]
* A pale grey death spirit pounds at you.  You repulse most of the fist with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in good position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit pounds at you.  You block with a damite targe.  
[You're nimbly balanced and in dominating position.]
You feel fully rested.
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced with no advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced with no advantage.]
* A pale grey death spirit pounds at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in dominating position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit claws at you.  You counter most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced with no advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>weave

You weave back and forth in a dance-like motion, drawing your opponent's attention away from the fight.
[You're nimbly balanced and in dominating position.]
Roundtime: 3 sec.
* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and overwhelming your opponent.]
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced and opponent has slight advantage.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
A spectral, darkly shimmering hydra rears high behind you, and with a discordant roar lashes out in an unbridled turmoil of thirteen long-necked heads thrashing about!

A cage of shadowy tendrils coalesces around a pale grey death spirit.
A cage of shadowy tendrils coalesces around a pale grey death spirit.
The mirage-hydra sinks into the ground.

* A pale grey death spirit claws at you.  You fend off most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced with no advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

* A pale grey death spirit claws at you.  You block solidly with a damite targe.  
[You're nimbly balanced and in superior position.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

[hunting-buddy: ***STATUS*** 106 minutes of hunting remaining or waiting on Large Edged, Staves, Tactics, Heavy Thrown, Brawling, Light Armor]

* A pale grey death spirit claws at you.  You dodge.  
[You're nimbly balanced and opponent has slight advantage.]
* A pale grey death spirit claws at you.  You fend off most of the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced with opponent in better position.]
You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
You feel fully attuned to the mana streams again.
A pale grey death spirit screams with a blood-curdling cry!

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]
A cage of shadowy tendrils coalesces around a pale grey death spirit.
A pale grey death spirit appears less distracted.
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced with no advantage.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>hunt

You take note of all the tracks in the area, so that you can hunt anything nearby down.

You were unable to locate any followable tracks.

Roundtime: 8 sec.
* A pale grey death spirit claws at you.  You counter nearly all the claw with a steel hhr'ata.  
[You're nimbly balanced and in good position.]
You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You beat off most of the claw with a steel hhr'ata.  
[You're nimbly balanced and have slight advantage.]
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced and in good position.]
* A pale grey death spirit slices wide at you.  You slap away most of the claw with a steel hhr'ata.  
[You're nimbly balanced and in strong position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You pick up a steel hhr'ata.

[combat-trainer]>bob

You bob suddenly, lowering yourself into a smaller target.
[You're nimbly balanced and in dominating position.]
Roundtime: 3 sec.
* A pale grey death spirit claws at you.  You fend off most of the claw with a steel hhr'ata.  
[You're nimbly balanced and in good position.]
* A pale grey death spirit claws at you.  You turn aside most of the claw with a steel hhr'ata.  
[You're nimbly balanced and in superior position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 3 sec.]
* A pale grey death spirit claws at you.  You evade.  
[You're nimbly balanced and in excellent position.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

You feel fully rested.
You feel fully attuned to the mana streams again.
* A pale grey death spirit claws at you.  You counter nearly all the claw with a platinum parry stick inlaid with bands of jade.  
[You're nimbly balanced and in strong position.]
You pick up a steel hhr'ata.

[combat-trainer]>perc

You reach out with your senses and see glowing (11/21) streams of golden Holy mana radiating through the area.
Letting your senses extend further, you feel there is shimmering (9/21) mana to the south, and luminous (14/21) mana to the north.
You sense the Centering spell upon you, which will last for about twenty-one roisaen.
You sense the Benediction spell upon you, which will last for about seventeen roisaen.
You sense your Osrel Meraud spell linking you to a luminous orb of coruscating rainbowed light, which should last until you release it.
You sense the Persistence of Mana spell upon you, which is linked to your Osrel Meraud orb.
You sense the Murrula's Flames spell upon you, which is linked to your Osrel Meraud orb.
You sense the Glythtide's Gift spell upon you, which is linked to your Osrel Meraud orb.
You sense the Major Physical Protection spell upon you, which will last for about sixteen roisaen.
You sense the Bless spell upon you, which will last for about fifteen roisaen.
You sense the Hydra Hex spell upon you, which will last until you fail to provide 15 mana for it.
You sense the Protection from Evil spell upon you, which will last for about nineteen roisaen.
Roundtime: 3 sec.
* A pale grey death spirit claws at you.  You block with a damite targe.  
[You're nimbly balanced and in better position.]

[combat-trainer]>lob

Your hhr'ata passes through the grey death spirit with no effect and falls to the ground!
[Roundtime 2 sec.]

[combat-trainer]>get my hhr'ata

What were you referring to?

[combat-trainer]>get hhr'ata

**Comment by rcuhljr** [comment]
Hopefully resolved. Open up a new issue if things go wrong again with bless.
